src/CONST.ts

```diff
@@ -1386,6 +1386,11 @@ const CONST = {
         },
         ID_FAKE: '_FAKE_',
         EMPTY: 'EMPTY',
+        MEMBERS_BULK_ACTION_TYPES: {
```

Comment:
Not sure that we need this, see my comments below about `handleBulkAction`

Reply:
yes, we still need this because we use these values also while creating options for ButtonWithDropdownMenu component